app/models/spree/variant_decorator.rb

```diff
@@ -55,6 +55,12 @@ def google_base_item_group_id
       product.slug
     end
 
+    def google_base_item_color 
+      if option_values
+        option_values.first.name if option_values.first
```

Comment:
i think you should be able to condense this, if a variant has `option_values` there will always be at least a first, right?

so maybe something like `return option_values.first.name if option_values` ?